openreview/venue_request/venue_request.py

```diff
@@ -185,6 +185,81 @@ def setup_review_stage(self):
             }
         ))
 
+    def setup_official_comment_stage(self):
+
+        official_comment_stage_content = {
+            'commentary_start_date': {
```

Comment:
don't we need an end date?

Reply:
@melisabok I have included this in the latest changes. The parameter is called `end_date` as it corresponds to the expiry date for comment invitations. I have applied start date and end date to both public and official comment invitations. Please take a look.